pie/scripts/group.py

```diff
@@ -125,5 +125,59 @@ def info(model_file):
     click.echo(repr(m))
 
 
+@pie_cli.command("finetune")
+@click.argument('input_model', type=click.Path(exists=True, file_okay=True, readable=True, dir_okay=False))
+@click.argument('train', type=click.Path(exists=True, file_okay=True, readable=True, dir_okay=False))
+@click.argument('dev', type=click.Path(exists=True, file_okay=True, readable=True, dir_okay=False))
+# @click.option('--test', type=str, default=None)
+@click.option('--seed', type=int, default=None)
+@click.option("--batch-size", type=int, default=None)
+@click.option("--epochs", type=int, default=100)
+@click.option("--mode", type=click.Choice(["expand", "skip", "replace"]), default="expand")
+@click.option("--device", type=str, default="cpu")
```

Comment:
It could help in avoiding mistakes and avoid typing one more option to choose the device automatically based on what's available (for the default value)